apps/hermes/server/src/api.rs

```diff
@@ -150,6 +150,10 @@ where
         .route("/api/get_price_feed", get(rest::get_price_feed))
         .route("/api/get_vaa", get(rest::get_vaa))
         .route("/api/get_vaa_ccip", get(rest::get_vaa_ccip))
+        .route(
```

Comment:
we have deprecated the old api endpoints and don't add things to them. could you please move it to `v2` and use something like `v2/updates/publisher_stake_caps/latest`?

Reply:
done